jaeger-core/src/main/java/com/uber/jaeger/senders/UDPSender.java

```diff
@@ -63,15 +69,15 @@ public UDPSender(String host, int port, int maxPacketSize) {
     memoryTransport = new AutoExpandingBufferWriteTransport(maxPacketSize, 2);
 
     udpTransport = TUDPTransport.NewTUDPClient(host, port);
-    udpClient = new Agent.Client(new TCompactProtocol(udpTransport));
-    maxSpanBytes = maxPacketSize - emitZipkinBatchOverhead;
-    spanBuffer = new ArrayList<Span>();
+    udpClient = new Agent.Client(new TBinaryProtocol(udpTransport));
```

Comment:
I think we should continue using compact protocol. The agent has a port that accepts jaeger-compact and jaeger-binary. The Binary is only used by the Node.js client, since thriftrw we're using there does not support Compact.